src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -883,6 +885,11 @@ async def Tokenize(
             offsets = None
 
             if request.return_offsets:
+                if is_mistral_tokenizer:
+                    raise ValueError(
+                        "Mistral tokenizer doesn't support "
+                        "return_offsets at the moment. "
+                    )
```

Comment:
Another small thought - probably better to move this above the tokenizer.encode above .. i.e. have an `if request.return_offsets: raise ValueError...` in that same `if mistral_tokenizer` block (so that it doesn't unnecessarily perform the tokenization in this case)

Reply:
good idea